packages/webapp/src/components/Forms/SimpleCustomType/index.jsx

```diff
@@ -93,19 +93,34 @@ const PureSimpleCustomType = ({
   const disabledButton = (!isValid || !isDirty) && !readonly;
   useHookFormPersist();
 
+  // Separating these into separate vs prop rendered nodes prevents form submission onClick for noSubmitButton
```

Comment:
Here we could use a similar approach to what I mentioned in the `GeneralRevenue` component.